tests/unit/helpers.py

```diff
@@ -3,41 +3,17 @@
 
 """Helper functions used to unit test Jenkins charm."""
 
-from pathlib import Path
-
 import requests
 
 
-def make_relative_to_path(tmp_path: Path, root_path: Path) -> Path:
-    """Make a root path (/path/from/root) relative to tmp_path.
-
-    If a path starts with "/", the path join operator "/" doesn't append the paths together but
-    replaces the path with the latter path. This helper function overrides that behavior.
-
-    Args:
-        tmp_path: Temporary path to base off of.
-        root_path: Root path beginning with /.
-
-    Returns:
-        Path relative to tmp_path.
-    """
-    if str(root_path).startswith("/"):
-        root_path = Path(str(root_path).replace("/", "", 1))
-    return tmp_path / root_path
-
-
+# There aren't enough public methods with this patch class.
 class ConnectionExceptionPatch:  # pylint: disable=too-few-public-methods
     """Class to raise ConnectionError exception."""
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self, *_, **__) -> None:
```

Comment:
I think it is fine to use `_` for both?

Reply:
<img width="472" alt="image" src="https://github.com/canonical/jenkins-k8s-operator/assets/37652070/933267db-cee6-4c75-a825-efb86e6a4fe7">
I'm getting a "duplicate parameter" error when using a `_` for both. :D It would be nice to be able to just keep all unused vars `_` though!

Reply:
Thats pylance being wrong 😉

Reply:
```
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/lint/lib/python3.10/site-packages/pydocstyle/parser.py", line 427, in parse
    compile(src, filename, 'exec')
  File "/home/yanks/Documents/canonical/jenkins-k8s-operator/tests/unit/test_jenkins.py", line 101
    def __init__(self, *_, **_) -> None:
                             ^
SyntaxError: duplicate argument '_' in function definition
```
Hm... I get a syntax error too when I try to run `tox` after the change. Would a variable name such as `unused` be preferred here? https://peps.python.org/pep-0640/

Reply:
I guess `*_args` and `**_kwargs` would also work

Reply:
And yes it seems Python doesn't allow duplicate input arguments
![Screenshot 2023-05-12 at 12 19 38 pm](https://github.com/canonical/jenkins-k8s-operator/assets/51036209/e0613079-8b8c-4d59-8545-1902c0733563)

Reply:
Changed! Thank you :D